sdk/appconfiguration/Azure.ApplicationModel.Configuration/src/ConfigurationClient.cs

```diff
@@ -462,6 +463,64 @@ public virtual Response<ConfigurationSetting> Get(string key, string label = def
             }
         }
 
+        public virtual async IAsyncEnumerable<ConfigurationSetting> GetSettingsAsync(SettingSelector selector, CancellationToken cancellationToken = default)
+        {
+            string nextLink = null;
```

Comment:
I think we can turn this into a helper method but I would like to see implementations for other clients first.